_posts/2017-07-29-markup-list-of-articles-with-h-feed.md

```diff
@@ -0,0 +1,11 @@
+---
+price: "$10"
+title: "Mark-up the list of articles on ScalaWilliam.com as h-feed"
+skills: [ html, indieweb, semantic, microformats ]
+layout: post
+---
+
+- Issue: [https://github.com/ScalaWilliam/ScalaWilliam.com/issues/64](https://github.com/ScalaWilliam/ScalaWilliam.com/issues/64)
+
+## Scope
```

Comment:
That’s fine.

I wanted to make sure the work post made clear that is involved more than just adding the mark-up and followed the only example of more content that you had: `2017-06-29-Play-Compile-DI-Application-Loader.md`.